### ISSUE TITLE
[3.1] User list is not HTML-compliant, breaks screen readers

### ISSUE THREAD

**Issue by prlanzarin** [bigbluebutton/bigbluebutton#23880]
**Describe the bug**

v3.1.x-release's user list is currently not HTML-compliant, which
breaks screen readers/acessibility.

This has previously been addressed in 3.0. The user list was rewritten
in 3.1, so it's likely the changes didn't have an effect when last merging
3.0 into 3.1.

See https://github.com/bigbluebutton/bigbluebutton/issues/23431 for the 3.0 reference - item 1, PR #23447.

**Expected behavior**

User list should be HTML compliant and work with screen readers.

**Actual behavior**

> The list currently is no valid html list, it is just one long string for
screenreaders.

**BBB version:**

v3.1.x-release up to https://github.com/bigbluebutton/bigbluebutton/commit/1001a806508ec83544a1758c02c2b897997306d5.

**Additional context**

https://github.com/bigbluebutton/bigbluebutton/issues/23431
